### PR TITLE
Code Quality: Cleanup code

### PR DESCRIPTION
**Pull request by Lamparter**
🧪 Built and tested using Visual Studio Batch Build!

### CODE REVIEW

File: src/FluentHub.Octokit/GlobalUsings.cs [hunk 1]
```diff
@@ -2,22 +2,20 @@
...
-global using global::System.Collections.Generic;
-global using global::System.Linq;
-global using global::System.Linq.Expressions;
-global using global::System.Runtime.Serialization;
-global using global::System.Text;
-global using global::System.Threading.Tasks;
-global using global::Newtonsoft.Json;
-global using global::Newtonsoft.Json.Converters;
-
 // Custom usings
```
Comment (Lamparter) [inline src/FluentHub.Octokit/GlobalUsings.cs:5]:
```suggestion
```

As you can see, doing this has messed up some notes surrounding import statements (see the comment on https://github.com/0x5bfa/FluentHub/pull/510/commits/d8b4c221751925ab7a432a19989149b9cdd5669a)

I'll just remove this comment and leave it in the pretty formatted way.

File: src/FluentHub.Octokit/GlobalUsings.cs [hunk 2]
```diff
@@ -2,22 +2,20 @@
 // Licensed under the MIT License. See the LICENSE.
 
 // Default usings
```
Comment (Lamparter) [inline src/FluentHub.Octokit/GlobalUsings.cs:4]:
```suggestion
```

File: src/FluentHub.Octokit/GlobalUsings.cs (hunk 1 above)
Comment (Lamparter) [reply src/FluentHub.Octokit/GlobalUsings.cs]: Fixed in [`beb88cd`](https://github.com/0x5bfa/FluentHub/pull/510/commits/beb88cdeb2f270f4330a014b23d1b468150a1e1a)

**Comment by Lamparter** [comment]
@0x5bfa I'll wait for your review :octocat:

File: src/FluentHub.App/GlobalUsings.cs
Comment (0x5bfa) [inline src/FluentHub.App/GlobalUsings.cs:1]: Can you revert changes here?

**Review by 0x5bfa — APPROVED** [review]
Otherwise, LGTM.
- Removing unnecessary using directives
- Formatting C# code

**Review by 0x5bfa — DISMISSED** [review]
As described above.

File: src/FluentHub.App/GlobalUsings.cs
Comment (Lamparter) [reply src/FluentHub.App/GlobalUsings.cs:1]: Done!

**Review by 0x5bfa — APPROVED** [review]

**Pull request merged**